include/felspar/parse/insert.detail.hpp

```diff
@@ -0,0 +1,25 @@
+#pragma once
+
+
+#include <span>
+
+
+namespace felspar::parse::binary::detail {
+
+
+    template<typename T>
+    inline void native_insert(
+            std::span<std::byte, sizeof(T)> const s, T const t) noexcept {
+        auto const addr = reinterpret_cast<std::byte const *>(&t);
```

Comment:
Again, aren't you supposed to tell the compiler when you're type-punning?  Would `std::as_bytes` help?

Reply:
Yeah, I've changed these now and made sure everything is consistent.